docs/data/sensitive-data/crypt4gh_client.md

```diff
@@ -1,187 +1,252 @@
 
-2) If you need to use data other environments too, can encrypt the data with one or several public keys. In this case, when imported in SD Desktop, you need to decrypt the data there manually. Note that in this case you need to have a corresponding secret key in SD Desktop. In practice this means that you have to encrypt your private key using the option 1 above and upload it to SD Desktop through SD Connect.
+   2) You can encrypt the data with your own key pair. In this case, when imported to SD Desktop, the data needs to be decrypted manually.
 
+**When using SD Connect to safely share (or transfer) data with your collaborators, you need to plan data encryption in advance, as you need to encrypt the data with your collaborator's public encryption key for them to be able to decrypt the data**. Using Crypt4GH CLI, it is possible to encrypt data with multiple public encryption keys. Thus, for example, the sama dataset can be safely shared with multiple colleagues or collaborators.
 
-* **When using SD Connect to safely share (or transfer) data with your collaborators, you need to plan data encryption in advance, as you need to encrypt the data with your collaborator's public encryption key for them to be able to decrypt the data**. Using Crypt4GH CLI, it is possible to encrypt data with multiple public encryption keys. Thus, for example, the same dataset can be safely shared with multiple colleagues or collaborators.
+!!! Note
+    Files that have been encrypted with the _CSC Sensitive Data Services public key_, can't be used in any other services as the corresponding secret key is available only to the SD services environment. If you wish to encrypt your data for some other service, you should create another encrypted file that uses other public keys.
 
+## Crypt4GH graphical user interface
 
-!!! Note
-    Files that have been encrypted only with the _CSC Sensitive Data Services public key_, can't be used in any other services as the corresponding secret key is             available only in the SD services environment. If you wish to encrypt your data for some other service, you should do another
-    encrypted file that uses other public keys.
+CSC has developed a simple graphical user interface (GUI) which will allow you to generate your encryption keys, encrypt your data, and to decrypt data using Crypt4GH.
 
+### Step 1: Install the Crypt4GH encryption GUI tool
 
+Download the version specific to your operating system from the [GitHub repository](https://github.com/CSCfi/crypt4gh-gui/releases):  
 
+- [Windows](https://kannu.csc.fi/s/iDiNR5HdwtFrXCY)
+- [Mac](https://kannu.csc.fi/s/88MFCb4wNRt2mwb)
+- [Linux](https://kannu.csc.fi/s/NAgiSeS8mFXKnC4)
 
+Please check that the tool has been digitally signed by CSC - IT Center for Science. After the download, you can find the Crypt4GH application in your downloads folder.
```

Comment:
we cannot state this for all OS, but only for Windows.

Reply:
Other OSes do not show verification dialog so the whole sentence is a moot point. But if a dialog appears, then it is necessary to verify the signature. Explaining this for every other step moves focus away from the instructions and quite some of our guides should then be written separately for macOS/Windows/Linux.

For simplicity, I'd leave this as is.